### ISSUE TITLE
Overlays on azaleas aren't visible.

### ISSUE THREAD

**Issue by Terminxa** [PepperCode1/Continuity#175]
It seems to only happen on that specific block. Heres comparison between OptiFine and Continuity (using Fabulously Optimized)
OptiFine:
![2022-10-10_18 27 07](https://user-images.githubusercontent.com/106455196/194961904-342fa569-2163-4909-9a83-ca282505ee47.png)
Continuity (Fabulously Optimized):
![2022-10-10_18 29 24](https://user-images.githubusercontent.com/106455196/194961924-c06c44d7-fd88-4390-921b-f229c5e2762d.png)

The resource pack I am using is JustTimm's Vanilla Additions.

**Comment by PepperCode1** [comment]
Is the block in the same place? If so, there is no guarantee that the random texture at a specific location with Continuity will be exactly the same as with Optifine because Continuity does not calculate which random texture to use exactly like Optifine. Try placing some more blocks.

**Comment by Terminxa** [comment]
> Is the block in the same place? If so, there is no guarantee that the random texture at a specific location with Continuity will be exactly the same as with Optifine because Continuity does not calculate which random texture to use exactly like Optifine. Try placing some more blocks.

yes, it is in the exact same place. it is like this for all flowering azaleas placed in the world. heres more comparison images:
optifine:
![image](https://user-images.githubusercontent.com/106455196/194964935-cea5d2de-8dbb-4e18-85e4-4865f05720ec.png)
continuity (fabulously optimized):
![image](https://user-images.githubusercontent.com/106455196/194964973-defbdc5c-53d5-429f-af21-6babe6cb745e.png)

**Comment by PepperCode1** [comment]
I tested this and the issue is actually that any sort of overlay does not render on azalea bushes. I am not sure why or how to fix it yet, but I will investigate further later. I suspect that the overlay quad is being rendered under the azalea quad and thus never appears.
